lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -739,6 +765,15 @@ export class BrowserCacheManager extends CacheManager {
         return null;
     }
 
+    /**
+     * Returns active account with tenant profiles if found 
+     * @returns 
+     */
+    getActiveAccountMultiTenant(): AccountInfo | null {
+        const activeAccount = this.getActiveAccount();
+        return activeAccount ? this.buildMultiTenantAccount(activeAccount) : null;
```

Comment:
Could this additional logic live in the existing getActiveAccount function?

Reply:
@tnorling It can, I wanted to avoid adding an optional parameter to the method signature, but I guess since it doesn't constitute a breaking change it's fine and probably better.

Reply:
Fixed, thanks!

Reply:
Doesn't look like those changes were pushed?